frontend/vre/field/record.annotations.view.js

```diff
@@ -2,13 +2,28 @@ import _ from 'lodash';
 import Backbone from 'backbone';
 import { AnnotationEditView } from '../annotation/annotation.edit.view';
 import { vreChannel } from '../radio.js';
-import { RecordFieldsBaseView } from './record.base.view';
+import {AggregateView} from "../core/view";
+import recordAnnotationsTemplate from "./record.annotations.view.mustache";
+import {CommentView} from "../annotation/comment.view";
+import {Annotation} from "../annotation/annotation.model";
 
-export var RecordAnnotationsView = RecordFieldsBaseView.extend({
-    title: 'Annotations',
+export var RecordAnnotationsView = AggregateView.extend({
+    template: recordAnnotationsTemplate,
+    container: 'tbody',
+
+    makeItem: function(model) {
+        var row = new CommentView({model: model});
+        row.on('edit', this.edit, this);
+        return row;
+    },
```

Comment:
If the `edit` event is emitted by the model instead of by the row view, you can just listen for `edit` events on `this.collection` and you don't need to bind to the event for every subview separately. That means the middle line of this method can be omitted. The remainder of the method is identical to the default definition, which means you can eliminate it entirely and just set `subview`:

```suggestion
    subview: CommentView,
```